cmd/bundle/bundle.go

```diff
@@ -21,5 +21,6 @@ func New() *cobra.Command {
 	cmd.AddCommand(newTestCommand())
 	cmd.AddCommand(newValidateCommand())
 	cmd.AddCommand(newInitCommand())
+	cmd.AddCommand(newGenerateCommand())
```

Comment:
Did we already settle on the name for this command? I saw many possible proposals in the original design?

I'd mainly prefer a command that we can also use to create new blank resources. So `generate` can work, kind of, but then we should use a form like `generate job --existing-job-id 1234`. This way it would also be possible to (perhaps in the future) just call `generate job`.

Reply:
@lennartkats-db there is no yet agreed upon name but the `generate` was the one that we leaned towards though subject for discussion and change still